float32 StepX
float32 StepY
---
int8 CanStep
